doc/hacking/bugs/bgo-650785/bisect.sh

```diff
@@ -1,10 +1,10 @@
 GOOD=/home/thomas/gst/git/gst-plugins-good/gst/audioparsers
-MORI=/home/thomas/dev/own/morituri
+WHIP=/home/thomas/dev/own/whipper
```

Comment:
Yeah, seems so.
I think that in the near future we'll probably `rm -r etc/bash_completion.d` (`doc/` replaced (?) and there's also lots of cruft in `misc/`).